api/src/middleware/publications.js

```diff
@@ -103,19 +103,18 @@ async function validateAuthorId(req, res, next) {
       ]),
     );
   }
-  await axios
-    .get(`https://scholar.google.com.sg/citations?user=${_id}`)
-    .then(() => { next(); })
-    .catch((error) => {
-      // if you mess around with the user id you only get 404
-      next(
-        fillErrorObject(
-          error.response.status,
-          'Validation error',
-          ['No Google Scholar user profile found with the given id'],
-        ),
-      );
-    });
+
+  try {
+    await axios(`${playwrightConfig.baseUrl}${_id}`);
+    next();
+  } catch (error) {
+    // if you mess around with the user id you only get 404
+    next(
```

Comment:
prefix with a `return` to return early from this handler and avoid executing any unintentional code if added below.